RiotSharpTest/ApiTestHelper.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+
+using Moq;
+using RiotSharp;
+using RiotSharp.ChampionEndpoint;
+using RiotSharp.SummonerEndpoint;
+using Newtonsoft.Json;
+
+namespace RiotSharpTest
+{
+    /// <summary>
+    /// Contains a variety of helper methods to generate objects for API testing
+    /// </summary>
+    static class ApiTestHelper
+    {
+        private static Summoner sum1 = new Summoner()
+        {
+            Id = 57039082,
+            Name = "keanefo",
+            Region = Region.na
+        };
+        private static Summoner sum2 = new Summoner()
+        {
+            Id = 31588136,
+            Name = "Zaganox",
+            Region = Region.na
+        };
+        private static Champion chp1 = new Champion()
+        {
+            Id = 1234,
+            FreeToPlay = true,
+            Active = true,
+            BotEnabled = true,
+            BotMmEnabled = true,
+            RankedPlayEnabled = true
+        };
+        private static Champion chp2 = new Champion()
+        {
+            Id = 3214,
+            FreeToPlay = false,
+            Active = false,
+            BotEnabled = false,
+            BotMmEnabled = false,
+            RankedPlayEnabled = false
+
+        };
+
+        private static Mastery mast1 = new Mastery()
+        {
+            Id = 12345,
+            Rank = 1
+        };
+
+        private static Mastery mast2 = new Mastery()
+        {
+            Id = 54321,
+            Rank = 2
+        };
+
+        private static MasteryPage mp1 = new MasteryPage()
+        {
+            Current = true,
+            Id = 1234,
+            Masteries = new List<Mastery>()
+            {
+                mast1,
+                mast2
+            },
+            Name = "Test"
+        };
+
+        private static MasteryPages pg1 = new MasteryPages()
+        {
+            Pages = new List<MasteryPage>()
+            {
+                mp1
+            },
+            SummonerId = 57039082
+        };
+
+        private static List<MasteryPages> masteryList = new List<MasteryPages>()
+        {
+
+        };
+
+
+        private static string team = "TEAM-66fbf960-a74c-11e4-a5a6-c81f66dd45c9";
```

Comment:
you can make all of these properties instead of having methods to get them all.  Methods can still be used for anything more complex than retrieval of the static data